arc-core/src/arc/util/OS.java

```diff
@@ -20,15 +20,11 @@
     static{
```

Comment:
if its android/ios, its definitely not gonna be windows
thats a small change and its kinda pointless, but whatever